.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: minimal
+
+services:
+  - docker
+
+env:
+  global:
+    - IMAGE_NAME_DOCKER_HUB="vshn/wrestic"
```

Comment:
I'd propose to change this to `IMAGE_NAME` and use the full image name (i.e. `docker.io/vshn/wrestic`).
Also we can use the `$TRAVIS_REPO_SLUG` [env var](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) to make the set up more generic:
`- IMAGE_NAME: docker.io/$TRAVIS_REPO_SLUG`